packages/react-lowcode/src/codegen/generation/generators/detail/mui-detail-generator.ts

```diff
@@ -107,18 +107,20 @@ export default class MuiDetailGenerator implements DetailGenerator {
               if (ts.isJsxAttribute(prop)) {
                 const propName = prop.name.escapedText.toString();
                 const inputProp = widgetProperties.properties.find(l => l.name === propName);
-
                 if (inputProp) {
                   if (prop.initializer) {
-                    if (ts.isStringLiteral(prop.initializer)) {
-                      if(inputProp.value !== prop.initializer.text) {
-                        newProp = factory.updateJsxAttribute(prop, prop.name, factory.createStringLiteral(inputProp.value));
-                        astChanged = true;
+                    const type = getTypeOfValue(inputProp.value)
```

Comment:
This is strange and buggy. You should use enum in WidgetProperty instead (you are trying to guess if it is really a expression from string value). If you need you can have function for validating expressions.

Reply:
We cannot use `WidgetProperty` enum because, this enum is set before changes have been made. This means if I change `string` value into `expression` value it will be wrapped into `""` thats why i need method to guess from new incoming value if it is `EXPRESSION` or `STRING_LITERAL`